src/diagonal.works/b6/cmd/b6/js/static/b6.css

```diff
@@ -464,6 +464,8 @@ form.shell-form > input {
 }
 
 .line {
+    width: 282px;
```

Comment:
Had a closer look at the line width thing, as the ghost border was happening again when scrollbars are always visible (windows behaviour), and also noticed that we have some lines that are wider than 282px.

<img width="200" alt="Screenshot 2024-03-22 at 10 24 25" src="https://github.com/diagonalworks/diagonal-b6/assets/23224854/44632cbf-7175-45e3-932c-bb666d0113bc">



I think it would make more sense that the width is defined by the stack and not by the line. So that inside the stack lines can have variable widths (e.g. if a scrollbar is present), but then all stacks have the same width.

So I suggest removing the three width definitions in `.line` and adding in .`.stack` a `width: 306px` property (as this seems to be the actual width that stacks with choice lines need for things not to get squished). 

Sort of related - I found this horizontal scroll going on, this can be sorted by removing the `width: 100%` from `.line-left-right-value`:
<img width="200" alt="Screenshot 2024-03-22 at 10 28 06" src="https://github.com/diagonalworks/diagonal-b6/assets/23224854/d23144dc-001c-44b5-8a08-4850d541ab41">

I've played around with these changes locally, so I could push a commit with these changes if that's easier.

Reply:
I merged in those changes. Thanks!